capstone/src/main/java/com/google/step/servlets/RestaurantsServlet.java

```diff
@@ -31,7 +31,7 @@
 import org.json.JSONObject;
 
 /** Servlet responsible for getting restaurants from Datastore. */
-@WebServlet("/restaurants")
+@WebServlet("/search")
```

Comment:
You may want to also update the line in script.js that calls this servlet onload to instead call /search now.

Reply:
Oh, yeah. I ended up doing this in the other PR because I also had to add a request parameter to the body.